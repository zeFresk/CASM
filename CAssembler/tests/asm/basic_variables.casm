.at 10
a: .word 004
b: .word 005
c: .word 006

.start 30
lda a
ldi b
add c
hrs 00
