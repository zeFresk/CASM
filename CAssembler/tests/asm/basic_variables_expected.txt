30
10: 004
11: 005
12: 006
30: 210
31: 311
32: 612
33: 900
